### PR TITLE
IDVA5-1251 [Registration- analysis] Cost Endpoint

### PR DESCRIPTION
**Pull request by rdas-ch**
__Short description outlining key changes/additions__

__JIRA Ticket Number__

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change
* [ ] Infrastructure change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by ttingle-ch — DISMISSED** [review]
LGTM

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -9,6 +9,7 @@ spring.main.allow-bean-definition-overriding=true
 spring.data.mongodb.uri=${MONGODB_URL:mongodb://127.0.0.1:27017/acsp_application}
 spring.data.mongodb.database=${MONGODB_DATABASE:acsp_application}
 spring.data.mongodb.auto-index-creation=true
+ACSP01_TYPES=credit-card
```
Comment (swalke-ch) [inline src/main/resources/application.properties:12]: Can this be renamed to ACSP01_PAYMENT_METHODS ?

File: src/main/java/uk/gov/companieshouse/acsp/service/CostService.java [hunk 2]
```diff
@@ -4,20 +4,24 @@
...
 import java.util.Collections;
+import java.util.List;
 
 @Service
 public class CostService {
 
     @Value("${ACSP01_COST}")
     private String costAmount;
+    @Value("${ACSP01_TYPES}")
+    private List<String> paymentTypes;
```
Comment (swalke-ch) [inline src/main/java/uk/gov/companieshouse/acsp/service/CostService.java:17]: Update the local variable as per below comment

File: src/main/java/uk/gov/companieshouse/acsp/service/CostService.java [hunk 3]
```diff
@@ -4,20 +4,24 @@
 import org.springframework.stereotype.Service;
 import uk.gov.companieshouse.api.model.payment.Cost;
 
+import java.util.ArrayList;
```
Comment (swalke-ch) [inline src/main/java/uk/gov/companieshouse/acsp/service/CostService.java:7]: Remove code smell
https://code-analysis.platform.aws.chdev.org/project/issues?id=uk.gov.companieshouse%3Aacsp-api&pullRequest=167&resolved=false&types=CODE_SMELL

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by swalke-ch — APPROVED** [review]
You need to add the new environment variable ACSP01_PAYMENT_METHODS to docker  compose file

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Pull request merged**